src/parallel/ForceHelper.h

```diff
@@ -2,6 +2,9 @@
 
 #include <variant>
 
+#include "particleContainer/ParticleContainer.h"
+#include "particleContainer/ParticleIterator.h"
```

Comment:
is this needed in the header? If yes remove it in the cpp otherwise remove it here

Reply:
I think I got a compilation error when it was not there in the header. So I will delete it in the cpp.